src/engine/ruleset/decoders/f5/f5-apm-own.yml

```diff
@@ -0,0 +1,451 @@
+  # July 2018/07/17 17:51:58 iqui low litani[3126]: 01490142: :itanimi: onoru: data
+  - check: event.provider==01490142
+    logpar:
+      - ~content: :<~>:<~/ignore/ ><~tmp.rsa.misc.log_session_id>:<~/ignore/ ><~tmp.rsa.internal.event_desc>
+    map:
+      - rsa.internal.messageid : $event.provider
+      - event.code : $event.provider
+      - rsa.misc.log_session_id : $~tmp.rsa.misc.log_session_id
+      - rsa.internal.event_desc : $~tmp.rsa.internal.event_desc
+
+  # 01490165
+  # January 2017/01/20 14:14:16 ptateve very-high miurerep: 01490165: :toccaec: Access profile: fugi initialized with configuration snapshot catalog: labo  
+  - check: event.provider==01490165
+    map:
+      - rsa.internal.messageid : $event.provider
+      - event.code : $event.provider
+
+  # 01490500
+  # May 2018/05/07 06:39:06 iumt medium nulapari[1973]: 01490500: :tsunt: rnat:oremi:ectobeat: New session from client IP 10.187.64.126 (ST=uasiarch/CC=Malor/C=boriosa) at VIP 10.47.99.72 Listener upt (Reputation=oremipsu)
+  - check: event.provider==01490500
+    logpar:
+      - ~content: :<~>:<~/ignore/ ><~>:<~>:<~tmp.rsa.misc.logsession_id>:<~/ignore/ >New session from client IP <source.ip> \(<~tmp.geodata>\) at <~> <destination.ip> <~>\(Reputation=<~tmp.rsa.misc.category>\)
+    map:
+      - ~tmp.slash : /
+      - ~tmp.geodata : +s_replace/$~tmp.slash/*
+      - ~tmp.geokv : +parse_kv/$~tmp.geodata/=/*/'/'
+      - destination.geo.city_name : $~tmp.geokv.C
+      - destination.geo.country_name : $~tmp.geokv.CC
+      - destination.geo.region_name : $~tmp.geokv.ST
+      - event.code : $event.provider
+      - related.ip : +a_append/$source.ip
+      - related.ip : +a_append/$destination.ip
+      - rsa.internal.messageid : $event.provider
+      - rsa.misc.category : $~tmp.rsa.misc.category
+      - rsa.misc.log_session_id : $~tmp.rsa.misc.log_session_id
+
+  # 01490501
+  # March 2019/03/03 10:33:06 omnisis very-high uptatema[7023]: 01490501: :stiaec: Cicero: ven
+  - check: event.provider==01490501
+    logpar:
+      - ~content: :<~>:<~/ignore/ ><~tmp.rsa.misc.log_session_id>:<~/ignore/ ><~tmp.rsa.internal.event_desc>
+    map:
+      - event.code : $event.provider
+      - rsa.internal.event_desc : $~tmp.rsa.internal.event_desc
+      - rsa.internal.messageid : $event.provider
+      - rsa.misc.log_session_id : $~tmp.rsa.misc.log_session_id
+
+  # 01490504
+  # February 2016/02/12 13:12:33 billoi medium orev[6153]: 01490504: :tatemU: deF: sist1803.mail.local can not be resolved.
+  - check: event.provider==01490504
+    logpar:
+      - ~content: :<~>:<~/ignore/ ><~tmp.rsa.misc.log_session_id>:<~/ignore/ ><~tmp.rsa.web.fqdn> <~>
+    map:
+      - event.code : $event.provider
+      - rsa.internal.event_desc : $~tmp.rsa.internal.event_desc
+      - rsa.internal.messageid : $event.provider
+      - rsa.misc.log_session_id : $~tmp.rsa.misc.log_session_id
+      - rsa.web.fqdn : $~tmp.rsa.web.fqdn
+      - related.hosts : +a_append/$rsa.web.fqdn
```

Comment:
There are some of these that are repeated many times, can't we map them all at the end of the decoder?